lib/Gitlab/Api/Deployments.php

```diff
@@ -0,0 +1,23 @@
+<?php namespace Gitlab\Api;
+
+class Deployments extends AbstractApi
+{
+    /**
+     * @param int $project_id
+     * @return mixed
+     */
+    public function all($project_id)
```

Comment:
Could use the `OptionsResolver` like in https://github.com/m4tthumphrey/php-gitlab-api/blob/master/lib/Gitlab/Api/Repositories.php#L13 to allow pagination?

Reply:
Done. I wasn't sure whether this had pagination so I just left it out. I also added `$parameter` in the docblock for `Repositories` used as example as it was missing.